addons/point_of_sale/static/src/js/models.js

```diff
@@ -1451,7 +1452,7 @@ export class PosGlobalState extends PosModel {
                 }
             }
         }
-        return _.uniq(mappedTaxes, (tax) => tax.id);
+        return uniqueBy(mappedTaxes, (tax) => tax.id);
```

Comment:
Looks like it is used only once. You can move the utils on top of this file after the imports.